data/api/client-server/relations.yaml

```diff
@@ -0,0 +1,170 @@
+  version: "1.0.0"
+host: localhost:8008
+schemes:
+  - https
+  - http
+basePath: /_matrix/client/v1
+consumes:
+  - application/json
+produces:
+  - application/json
+securityDefinitions:
+  $ref: definitions/security.yaml
+paths:
+  "/rooms/{roomId}/relations/{eventId}/{relType}/{eventType}":
+    get:
+      summary: Get the child events for a given parent event, with optional filter.
+      description: |-
+        Retrieve all of the child events for a given parent event, optionally filtering
+        down by relationship type and related event type.
+
+        Note that while this endpoint indicates `relType` and `eventType` being
+        required, they can be individually omitted to retrieve all matching events.
+        When omitting the path parameters, a trailing slash must not be included
+        (otherwise the search will be for an empty string).
+
+        When combining the use of `relType` and `eventType`, the server will return
+        child events which match *both* conditions rather than *either*.
+      operationId: getChildEventsForParentEvent
+      security:
+        - accessToken: []
+      parameters:
+        - in: path
+          type: string
+          name: roomId
+          description: The ID of the room the supposed parent event is in.
+          required: true
+          x-example: "!636q39766251:matrix.org"
+        - in: path
+          type: string
+          name: eventId
+          description: The supposed parent event ID.
+          required: true
+          x-example: "$asfDuShaf7Gafaw"
+        - in: path
+          type: string
+          name: relType
+          description: |-
+            The [relationship type](/client-server-api/#relationship-types) to search
+            for, if any. Exclude to find all child events which point to the supposed parent
+            event with any `rel_type`.
+          required: true # we can't say false, annoyingly
+          x-example: "org.example.my_relation"
+        - in: path
+          type: string
+          name: eventType
+          description: |-
+            The event type of child events to search for, if any. Exclude to find all
+            child events of any type which point to the supposed parent event.
+
+            Note that in encrypted rooms this will typically always be `m.room.encrypted`
+            regardless of the event type contained within the encrypted payload.
+          required: true # we can't say false, annoyingly
+          x-example: "m.room.message"
+        - in: query
+          type: string
+          name: from
+          description: |-
+            The pagination token to start returning results from. If not supplied, results
+            start at the earliest topological event known to the server.
+
+            Can be a `next_batch` or `prev_batch` token from a previous call, or an equivalent
+            token from [`/messages`](/client-server-api/#get_matrixclientv3roomsroomidmessages)
```

Comment:
The [spec](https://spec.matrix.org/unstable/client-server-api/#get_matrixclientv3roomsroomidmessages) for `/messages` has this to say about its `from` param:

> This endpoint [sic] can also accept a value returned as a `start` token by a previous request to this endpoint, though servers are not required to support this. Clients should not rely on the behaviour.

... which seems to say that  a `start` from a previous `/messages` response is a thing that we're doing our best to deprecate, rather than allow to propagate?

In any case, I think it would be good to be explicit about what constitutes an "equivalent token".

Reply:
done so